internal/service/ipam/model_ipamsvc_range.go

```diff
@@ -95,6 +96,9 @@ var IpamsvcRangeResourceSchemaAttributes = map[string]schema.Attribute{
 		},
 		Optional:            true,
 		MarkdownDescription: "The list of DHCP options. May be either a specific option or a group of options.",
+		PlanModifiers: []planmodifier.List{
+			listplanmodifier.RequiresReplaceIfConfigured(),
+		},
```

Comment:
Why did you add this? This will basically delete the range and recreate it every time the `dhcp_options` is modified.

Reply:
DHCP option has 2 types, we can either create via option & group. we have added both in function steps and the update was not working when steps were creating DHCP option via option type and then update group type.